dushimeemma/kigc-esas-frontend
# #177960354 Allow admin to clear all records

#### What does this PR do?
- Allow admin to clear all records
#### Description of Task to be completed?
- Create a clear button
- Make actions to clear rooms, students, and courses
- Integrate the functionality with the backend
#### How should this be manually tested?
- Clone this repo and checkout the branch
- Install all dependencies and start the project
- Login and hit on one among the three (student, rooms, courses) found in the aside menu
> You should all records and red button in the right bottom corner of the window
> Clicking on it should clear all records found thus far
#### Any background context you want to provide?
Credentials used for login should be one for super admin
#### What are the relevant pivotal tracker stories?
[#177960354](https://www.pivotaltracker.com/story/show/177960354)
#### Screenshots (if appropriate)
![image](https://user-images.githubusercontent.com/44426710/116779772-a2428d00-aa78-11eb-9c65-5526d59a51d3.png)
